cpapshare.rb

```diff
@@ -123,11 +302,11 @@ def recursive_copy(source, destination)
   FileUtils.chmod(0o644, POLKIT_RULE_PATH)
   puts "Polkit rule installed to #{POLKIT_RULE_PATH}"
   # Reload polkit rules
-  warn 'Warning: Failed to reload polkit rules' unless system('systemctl reload polkit')
+  warn 'Warning: Failed to restart polkit' unless system('systemctl restart polkit')
 
 elsif opts[:uninstall]
-  File.delete(POLKIT_RULE_PATH)
-  puts "Polkit rule uninstalled from #{POLKIT_RULE_PATH}"
+  File.delete(POLKIT_RULE_PATH) if File.exist?(POLKIT_RULE_PATH)
+  puts "Polkit rule uninstalled from #{POLKIT_RULE_PATH}" if File.exist?(POLKIT_RULE_PATH)
```

Comment:
The second puts will never show up since the file was just deleted on previous line.